### PR TITLE
add NetwrixAuditor connector

### PR DESCRIPTION
**Pull request by KateShapovalova**
Required items, please complete
   
   Change(s):
   - See guidance below

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Solutions/NetwrixAuditor/Parsers/NetwrixAuditor.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                , SrcHostname=SourceHostName
+                , EventResult= Activity
+                , Object = FilePath
+                , ActorUsername = SourceUserName
+| mv-apply AdditionalFields = extract_all(@'(?P<key>[a-zA-Z0-9-_]+)=(?P<value>[a-zA-Z0-9-_:/@.,#{}><" ]+)', dynamic(["key","value"]), AdditionalExtensions) on (
+    project packed1 = pack(tostring(AdditionalFields[0]), tostring(AdditionalFields[1]))
+    | summarize bag = make_bag(packed1)
+)
+| evaluate bag_unpack(bag)
+| extend EventEndTime = todatetime(start)
```
Comment (manishkumar1991) [inline Solutions/NetwrixAuditor/Parsers/NetwrixAuditor.txt:26]: @KateShapovalova : Please take care of recent CEF schema changes , for start the new column name is <html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 15">
<meta name=Originator content="Microsoft Word 15">
<link rel=File-List
href="file:///C:/Users/MANISK~1/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

<link rel=themeData
href="file:///C:/Users/MANISK~1/AppData/Local/Temp/msohtmlclip1/01/clip_themedata.thmx">
<link rel=colorSchemeMapping
href="file:///C:/Users/MANISK~1/AppData/Local/Temp/msohtmlclip1/01/clip_colorschememapping.xml">

<style>

</style>

</head>

<body lang=EN-IN style='tab-interval:36.0pt;word-wrap:break-word'>




StartTime
--





</body>

</html>

Comment (manishkumar1991) [inline Solutions/NetwrixAuditor/Parsers/NetwrixAuditor.txt:26]: written statement has two issues:

1>why the start field is getting assigned to endtime ?

2> @KateShapovalova  Please take care of new cef schema changes while writing the parser , in new cef schema "start" is changed to StartTime 
Try to make the parser compatible for older schema as well new schema 

for Example : EventStartTime = coalesce(column_ifexists("StartTime",datetime(null)),todatetime(start))

File: Solutions/NetwrixAuditor/Parsers/NetwrixAuditor.txt [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+| extend EventOriginalResultDetails = cat
```
Comment (manishkumar1991) [inline Solutions/NetwrixAuditor/Parsers/NetwrixAuditor.txt:27]: @KateShapovalova  Consider new cef schema changes as well cat changed to DeviceEventCategory .
Try to make the parser compatible for older schema as well new schema
for example : EventOriginalResultDetails= coalesce(column_ifexists("DeviceEventCategory",""),cat)

Comment (KateShapovalova) [reply Solutions/NetwrixAuditor/Parsers/NetwrixAuditor.txt]: @manishkumar1991, done

**Comment by manishkumar1991** [comment]
@KateShapovalova  ,

Kindly check below screenshot , As per my thinking , Its a cef based data connector so data type should be commaneventformat

and if there is not custom log then NetwrixAuditor_CL should be changed to NetwrixAuditor

image

image

**Comment by KateShapovalova** [comment]
@manishkumar1991, done

**Review by devikamehra — APPROVED** [review]
All requested changes are completed now.

**Review by v-sabiraj — APPROVED** [review]

**Pull request merged**